README.md

```diff
@@ -302,6 +302,11 @@ params.SplitCount = 2
 params.MinSplitSize = 7168
 
 reader, totalDownloaded, totalExpected, err := rtManager.DownloadFilesWithResultReader(params)
+
+// Iterate over each file
+for currentFileInfo := new(utils.FileInfo); reader.NextRecord(currentFileInfo) == nil; currentFileInfo = new(utils.FileInfo) {
```

Comment:
Shouldn't ```== nil``` be changed to ```!= nil```?

Reply:
No, since `NextRecord()` returns an error.